app/assets/stylesheets/gto/_nav.scss

```diff
@@ -84,6 +84,15 @@
     list-style: none outside none;
     margin: 0;
     padding: 0;
+
+    .nav-secondary-link {
+      font-size: 0.75em;
```

Comment:
`em` here is set so it will be proportional to the parent font-size, since we are nesting styles. This way there will be no need to change it again within media queries. But if that's what the codebase standard is, I can change it to `rem`.

Reply:
if the parent is `rem` does that still work?  this project doesn't really follow the ideal but my goal would be that the base font size is `1px` and everthing (*EVERYTHING*) is `rem`s.  but i guess i don't know how `rem` as a child vs `ems` as a child would be different.  It sounds like you understand and `em` is probably right here.  and since we're not all the way to `all rems`, let's leave it as written => `em`